naples/models/user.py

```diff
@@ -59,7 +60,16 @@ class User(db.Model, ModelMixin):
 
     avatar: orm.Mapped["File"] = orm.relationship()
 
-    subscription: orm.Mapped["Subscription"] = orm.relationship(viewonly=True, back_populates="user")
+    subscriptions: orm.Mapped[list["Subscription"]] = orm.relationship(viewonly=True)
+
+    @property
+    def customer_stripe_id(self):
+        return self.subscriptions[0].customer_stripe_id if self.subscriptions else ""
+
+    @property
+    def subscription(self):
+        # get last saved data in subscriptions
+        return self.subscriptions[-1] if self.subscriptions else None
```

Comment:
Why do we use different subscriptions here? I suggest we always use the same subscription to get data from . And I suggest it to be the last subscription user has

Reply:
I agree, I'll fix it